### ISSUE TITLE
[12.0.0+] Cursor moves  to the next row when press ENTER to confirm on Japanese IME (version 12 only)

### ISSUE THREAD

**Issue by takyam-git** [handsontable/handsontable#9586]
#### Handsontable version 11
https://user-images.githubusercontent.com/471113/174003814-bd700546-cde0-494d-ae5d-2111c363813f.mp4

- keypress `aiueo[ENTER]`
- IME word selection is confirmed
- Cursor in `B2`

#### Handsontable version 12

https://user-images.githubusercontent.com/471113/174003870-90d5f983-5565-444c-81a2-127561ae0a95.mp4

- keypress `aiueo[ENTER]`
- IME word selection is confirmed
- Cursor move to `B3` (next row)

### Steps to reproduce

1. use Japanese IME
2. select cell and input with IME
3. press ENTER to confirm

### Demo

```js
import Handsontable from "handsontable";

const data = [
  ['', 'Tesla', 'Volvo', 'Toyota', 'Ford'],
  ['2019', 10, 11, 12, 13],
  ['2020', 20, 11, 14, 13],
  ['2021', 30, 15, 12, 13]
];

const container = document.getElementById('app');
const hot = new Handsontable(container, {
  data: data,
  rowHeaders: true,
  colHeaders: true
});
```

### Your environment
* Handsontable version: 12.0.1
* Browser name and version: Chrome 102
* Operating system: Mac Monterey

**Comment by adrianszymanski89** [comment]
Hi @takyam-git 

Thank you for reporting the issue. I tested it and indeed this functionality isn't working correctly. We introduced a new shortcut manager in v. 12.0.0 and that might be the reason. I will label that issue as a regression.

**Comment by adrianszymanski89** [comment]
Inform ZD 29519.

**Comment by adrianszymanski89** [comment]
Inform ZD 29816.

**Comment by adrianszymanski89** [comment]
Inform [Forum](https://forum.handsontable.com/t/japanese-text-on-enter/6432)

**Comment by aninde** [comment]
This issue will be fixed by the upcoming v12.1.3 https://jsfiddle.net/k7qLwdat/　
On the recording, after confirming with <kbd>Enter</kbd>, the selection does not move down.

https://user-images.githubusercontent.com/30186818/191247869-88b80ad2-db78-4260-8ea7-9cd1b34e8c4d.mov

**Comment by AMBudnik** [comment]
Hi @takyam-git 

I'm more than happy to notify you that we just released Handsontable v12.1.3 where this issue is fixed.

I'm closing this issue as solved. However, please feel free to leave a comment if needed.
